src/export/export.js

```diff
@@ -0,0 +1,487 @@
+                            <img src="${mapImagesToState(
+                              SLEEP.icon,
+                            )}" class="journal__item-image" />
+                            <p class="journal__item-symptom-name">${
+                              displayedCategories['SLEEP']
+                            }</p>
+                          </li>
+                        `
+                        }
+                        ${
+                          !NOTES
+                            ? ''
+                            : `
+                          <li class="journal__item-symptom-wrapper">
+                            <hr class="journal__note-separator" />
+                            <img src="https://cdn.iconscout.com/icon/premium/png-256-thumb/note-pencile-memo-pen-notebook-book-write-3-14650.png" class="journal__item-image" />
+                            <p class="journal__item-symptom-name">${NOTES}</p>
+                          </li>
+                        `
+                        }
+                      </ul>
+                    </div>
+                  </li>
+                `;
+              })
+              .join('')}
+            </ul>
+      </body>
+    </html>`;
+  };
+
+  const exportData = async () => {
+    await AsyncStorage.setItem(MailStorageKey, mail);
+    const htmlExport = formatHtmlTable();
+
+    setLoading(true);
+    const res = await fetch('https://api.tipimail.com/v1/messages/send', {
+      method: 'POST',
+      headers: {
+        'X-Tipimail-ApiUser': API_USER,
+        'X-Tipimail-ApiKey': API_KEY,
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        apiKey: API_KEY,
+        to: [
+          {
+            address: mail,
+          },
+        ],
+        msg: {
+          from: {
+            address: mailFrom,
+            personalName: mailFromName,
+          },
+          subject: 'Export de données',
+          html: htmlExport,
+        },
+      }),
+    }).catch((err) => console.log('sendNPS err', err));
+
+    setLoading(false);
+    if (res.ok) {
+      Alert.alert(
+        'Mail envoyé !',
+        `Retrouvez vos données sur votre boîte mail : ${mail}`,
+      );
+    } else {
+      Alert.alert("Une erreur s'est produite !");
+    }
+  };
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <TouchableOpacity
+        onPress={this.onBackPress}
+        style={styles.backButtonContainer}>
+        <Text style={styles.backButton}>{'Retour'}</Text>
+      </TouchableOpacity>
+      <ExportData style={styles.icon} />
+      {true && (
+        <Text style={styles.title}>
+          Recevez vos données des 30 derniers jours par mail
+        </Text>
+      )}
+
+      {false && (
+        <>
+          <Text style={styles.title}>Recevez vos données des</Text>
+          <TextInput
+            keyboardType="decimal-pad"
+            onChangeText={(txt) => setDays(parseInt(txt))}
```

Comment:
```suggestion
            onChangeText={(text => setDays(parseInt(text))}
```

+ Ajouter une fonction pour vérifier que le nombre entré n'est pas négatif ou trop grand